schema.md

```diff
@@ -0,0 +1,555 @@
+version, you should consider mentioning the upstream project in this
+key.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+## Section `legal`
+
+### Key `legal/license`
+
+* Type: string
+* Presence: mandatory
+* Example: `"AGPL-3.0-or-later"`
+
+This string describes the license under which the software is
+distributed. The string must contain a valid SPDX expression, referring
+to one (or multiple) open-source license. Please refer to the [SPDX
+documentation](https://spdx.org/licenses/) for further information.
+
+### Key `legal/main-copyright-owner`
+
+* Type: string
+* Presence: optional
+* Example: "City of Amsterdam and many contributors"
+
+This string describes the entity that owns the copyright on
+"most" of the code in the repository. Normally, this is the line
+that is reported with the copyright symbol at the top of most files
+in the repo.
+
+It is possible to list multiple owners if required so, using an English
+sentence. It is also possible to informally refer to a community of
+group of people like "Linus Torvalds and all Linux contributors".
+
+In case it is not possible to name a main copyright owner, it is
+possible to omit this key; in those cases, if the repo has a authors
+file, you can point to it through `legal/authors_file`.
+
+### Key `legal/authors-file`
+
+* Type: string (path to file)
+* Presence: optional
+* Example: "doc/AUTHORS.txt"
+
+Some open-source softwares adopt a convention of identify the copyright
+holders through a file that lists all the entities that own the
+copyright. This is common in projects strongly backed by a community
+where there are many external contributors and no clear single/main
+copyright owner. In such cases, this key can be used to refer to the
+authors file, using a path relative to the root of the repository.
+
+### Key `legal/repo-owner`
+
+* Type: string
+* Presence: mandatory
+* Example: "City of Amsterday"
+
+This string describes the entity that owns this repository; this might
+or might not be the same entity who owns the copyright on the code
+itself. For instance, in case of a fork of the original software, the
+`repo-owner` is probably different from the `main-copyright-owner`.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+## Section `maintenance`
+
+This section provides information on the maintenance status of the
+software, useful to evaluate whether the software is actively
+developed or not.
+
+### Key `maintenance/type`
+
+* Type: enumerate
+* Presence: mandatory
+* Values: "commercial", "community", "none"
+
+This key describes how the software is currently maintained.
+"Commercial" means that there is a commercial contract that binds
+an entity to the maintenance of the software; "community" means that
+the software is currently maintained by one or more people that donate
+their time to the project; "none" means that the software is not
+actively maintained.
+
+### Key `maintenance/until`
+
+* Type: date (YYYY-MM-DD)
+* Presence: mandatory (if the software is commercially maintained)
```

Comment:
I would say mandatory for any project that is budgetted, ideally we can also see from PA software when the budget is running out.

Reply:
I tried a different approach:

> In case of a maintenance, this key must contain the date
at which the maintenance is going to end. In case of community
maintenance, the value should not be more than 2 years in the
future, and thus will need to be regularly updated as the
community continues working on the project.